Common/src/main/resources/default_configs/example.json

```diff
@@ -0,0 +1,30 @@
+{
+  "mod_id": "", #add mod_id
+  "provided_stones": [
+    {
+	  "id": "modid_itemid", #Don't add ":"
+	  "block_id": "modid:itemid", #e.g. enriched:dark_granite
+	  "en_name": "", #Actual name, e.g. "Dark Granite
+      "types": [
+        "stone" #stone=overworld, end=end,nether=nether. Add the following for the dimension the blocks will spawn in

```

Comment:
It has nothing to do with the dimension; it is a category for the stone that associates ores with stones.